packages/design-system/src/components/DateField/MultiInputDateField.stories.jsx

```diff
@@ -7,11 +7,17 @@ export default {
   title: 'Components/MultiInputDateField',
   component: MultiInputDateField,
   argTypes: {
-    label: { control: false },
     errorMessage: {
       control: { type: 'text' },
       type: { name: 'string', required: true },
     },
+    label: { control: 'text' },
+    monthDefaultValue: { control: { type: 'number', min: 1, max: 12, steps: 1 } },
+    dayDefaultValue: { control: { type: 'number', min: 1, max: 31, steps: 1 } },
+    yearDefaultValue: { control: 'number' },
```

Comment:
I think in order for setting these to do anything, we need to update the story itself to no longer `useState` but `useArgs`. I've had mixed success with `useArgs`.

Reply:
Oh interesting I didn't realize that didn't update in that case.. I'm just going to remove these then, I don't think they add a lot of value, mostly just wanted to play with the min/max value stepper.